index.html

```diff
@@ -252,24 +271,29 @@ <h1 class="head3">Skills</h1>
                 </div>
             </div>
             <div class="contact-form">
-                <p class="contact-form-text">
-                    I'm always interested in hearing about new projects,
-                    so if you'd like to chat please get in touch.
-                </p>
-                <form class="text-form"  action="https://formspree.io/f/xbjwvddv" method="POST">
-                    <input type="text" name="name" placeholder="Full Name" maxlength="30" required>
-                    <input type="email" name="email" placeholder="Email Address" required>
-                    <textarea cols="30" rows="10" name="message" maxlength="500" placeholder="Enter text here"></textarea>
-                    <div class="btn"><button type="submit" class="contact-form-button">Get in touch</button></div>
-                    <hr>
-                    <div class="footer">
-                        <img src="./images/SocialMedia.png" alt="social media icons">
-                    </div>
-                </form>
+                <div class="contact-form-text">
+                    <p>
+                        I'm always interested in hearing about new projects,
+                        so if you'd like to chat please get in touch.
+                    </p>
+                </div>
```

Comment:
- [ ] Please kindly align your text, your design should stick as much as possible to the Figma design.

Your design

![Screenshot from 2022-07-08 23-30-51](https://user-images.githubusercontent.com/74501523/178074106-cd8e251f-51f1-40c8-812b-6d9b0cb83309.png)

Figma design

![Screenshot from 2022-07-08 23-31-18](https://user-images.githubusercontent.com/74501523/178074129-26a5b125-fdd4-41a0-838e-874aaca47bd2.png)